### ISSUE TITLE
effect goes away when window is inactive 

### ISSUE THREAD

**Issue by FayedFahad** [Maplespe/ExplorerBlurMica#134]
https://github.com/Maplespe/ExplorerBlurMica/assets/63035564/1061e7bf-a53d-48aa-9c67-38d6a5f4fe26

can this be fixed? help please

**Comment by Maplespe** [comment]
This is the normal effect of mica, not a bug.
If you want to customize the behavior and color of the mica effect, please use DWMBlurGlass with it.

**Comment by FayedFahad** [comment]
isn't DWMBlurGlass only used for titlebar effects?

**Comment by Maplespe** [comment]
You can use it with ExplorerBlurMica, just check "`Override DWMAPI mica effect (win11)`" in DWMBlurGlass to replace EBM's blur effect.

**Comment by FayedFahad** [comment]
can i get the exact same effect shown on the video with DWMBlurglass?

**Comment by Maplespe** [comment]
Yes, DWMBlurGlass allows you to adjust effects, colors, and even animation duration.
